tests/integration-tests/README.md

```diff
@@ -0,0 +1,110 @@
+# Integration Tests
+
+This directory contains end-to-end integration tests for the application using Playwright.
+
+## Prerequisites
+
+- Node.js (v16 or higher)
```

Comment:
We should be using Node >= v18. Anything older is unsupported https://nodejs.org/en/about/previous-releases

Reply:
Updated